### PR TITLE
htp: do not log content-encoding: none

### PR DESCRIPTION
**Pull request by catenacyber**
Ticket: #5757

https://redmine.openinfosecfoundation.org/issues/5757

### CODE REVIEW

File: htp/htp_transaction.c [hunk 1]
```diff
@@ -1453,7 +1453,7 @@ htp_status_t htp_tx_state_response_headers(htp_tx_t *tx) {
                                 "Compression bomb: multiple encoding with lzma");
                         break;
                     }
-                } else if (bstr_util_cmp_mem(tok, tok_len, "inflate", 7) == 0) {
+                } else if (bstr_util_cmp_mem(tok, tok_len, "inflate", 7) == 0 || bstr_util_cmp_mem(tok, tok_len, "none", 4) == 0) {
```
Comment (victorjulien) [inline htp/htp_transaction.c:1456]: should these comparisons be stricter? I seems like "infateme" and "noneother" would match here as well due to the limited length check

**Comment by victorjulien** [comment]
Merged in #384, thanks!

File: htp/htp_transaction.c (hunk 1 above)
Comment (catenacyber) [reply htp/htp_transaction.c:1456]: Do you want to ?

Comment (victorjulien) [reply htp/htp_transaction.c:1456]: Wonder if it can lead to impedance mismatch between client and suricata? Suri thinks lzma, client something else?

Comment (catenacyber) [reply htp/htp_transaction.c:1456]: I bet something can be crafted, but not easily

**Pull request closed without merging**